src/Compilers/CSharp/Test/Symbol/Compilation/UsedAssembliesTests.cs

```diff
@@ -3732,7 +3732,6 @@ public unsafe static void Main()
 }
 ");
 
-            // Support for `using unsafe static` tracked by https://github.com/dotnet/roslyn/issues/67329
```

Comment:
The comment was about the `using static C1<S<C0>*[]>;` scenario. Do we have the corresponding test scenario with "unsafe" added?
Same question for `using static C1<S<C0>*[]>.E1;` below.